test/ClangImporter/experimental_diagnostics_cmacros.swift

```diff
@@ -0,0 +1,166 @@
+// RUN: not %target-swift-frontend(mock-sdk: %clang-importer-sdk) -enable-experimental-clang-importer-diagnostics -typecheck %s 2>&1 | %FileCheck %s --strict-whitespace
+
+import macros
+
+_ = INVALID_INTEGER_LITERAL_2
+// CHECK:      macros.h:{{[0-9]+}}:{{[0-9]+}}: error: invalid digit 'a' in decimal constant
+// CHECK-NEXT:      #define INVALID_INTEGER_LITERAL_2 10abc
+// CHECK-NEXT: {{^}}                                    ^
+// CHECK-NEXT: macros.h:{{[0-9]+}}:9: error: macro 'INVALID_INTEGER_LITERAL_2' not imported
+// CHECK-NEXT:      #define INVALID_INTEGER_LITERAL_2 10abc
+// CHECK-NEXT: {{^}}        ^
+// CHECK-NEXT: macros.h:{{[0-9]+}}:35: note: invalid numeric literal
+// CHECK-NEXT:      #define INVALID_INTEGER_LITERAL_2 10abc
+// CHECK-NEXT: {{^}}                                  ^
```

Comment:
It is because "invalid digit 'a' in decimal constant" is not produced by me and is an error produced deeper in the tree, I'll have to double check where.

Reply:
OK. I didn't noticed that was not part of the new ones. No need to change, as I said. I simply noticed it and I was interested.